ext/spl/spl_fixedarray.c

```diff
@@ -561,6 +561,13 @@ SPL_METHOD(SplFixedArray, __construct)
 	spl_fixedarray_object *intern;
 	zend_long size = 0;
 
+	int rv;
+	zend_error_handling zeh;
+
+//	zend_replace_error_handling(EH_THROW, spl_ce_InvalidArgumentException, &zeh TSRMLS_CC);
+//	rv = zend_parse_parameters(ZEND_NUM_ARGS(), "|l", &size);
+//	zend_restore_error_handling(&zeh TSRMLS_CC);
```

Comment:
Presumably I made a mistake. This behaviour isn't covered by any test. I'll add a test for all of the SPL constructors tomorrow.